lib/api/api.go

```diff
@@ -499,6 +519,16 @@ func (s *service) CommitConfiguration(from, to config.Configuration) bool {
 	return true
 }
 
+func (s *service) IsAuthEnabled() bool {
```

Comment:
I don't see any external uses that require this to be exported?

Reply:
That's probably true, this is probably a relic from when this was moved out from being a method on `GUIConfiguration`.

Reply:
Fixed by https://github.com/emlun/syncthing/pull/4